api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
The difference between mentions and markdown links is mainly for user experience. Currently, the API strips any html out of a comment so in order to have the dropdown list of contributors that inserts an html element, the html needs to be converted to markdown prior to saving (the comments are rendered markdown). The conversion from html to markdown also prevents users from thinking they can mention non-contributors. However, there has been discussion of removing the html bleaching from the API which would reduce the complexity of the conversions in `comment.js` (which are not ideal).